vpr/SRC/route/route_timing.c

```diff
@@ -594,33 +722,26 @@ bool timing_driven_route_net(int inet, int itry, float pres_fac, float max_criti
 		}
 
 		// reexplore route tree from root to add any new nodes (buildheap afterwards)
-#ifdef PROFILE
-			clock_t route_tree_start = clock();
-#endif
-		add_route_tree_to_heap(rt_root, target_node, target_criticality,
-				astar_fac);
+
+		add_route_tree_to_heap(rt_root, target_node, target_criticality, astar_fac);
 		heap_::build_heap();	// via sifting down everything
-		// if (itarget == num_sinks && itarget > 800) {
-		// 	vpr_printf_info("heap for target %d: ", itarget); 
-		// 	heap_::verify_extract_top();
-		// }
-#ifdef PROFILE
-			build_heap_time += static_cast<float>(clock() - route_tree_start) / CLOCKS_PER_SEC;
-#endif
-		
+
+		INCREMENTAL_REROUTING_ASSERT(heap_::is_valid());
+
 
 		// cheapest s_heap (gives index to rr_node) in current route tree to be expanded on
-		struct s_heap* cheapest = get_heap_head();
+		struct s_heap* cheapest {get_heap_head()};

```

Comment:
I'm still not sure about the {} initialization syntax.  I find using = is clearer.